### PR TITLE
Add new mobs to allowed teleportable entities

### LINKED ISSUE #11: Add new entities from 1.14
> Check if any new entities were added in this new version of the API. Just commented out those no longer present.

from #10

### PR DESCRIPTION
**Pull request by zeroeightysix**
This PR adds 1.13 and 1.14 mobs to allowed teleportable entities and removes commented out mentions of `EntityType`s that no longer exist.

Closes #11

### CODE REVIEW

File: src/PortalUtils.java [hunk 1]
```diff
@@ -181,16 +182,17 @@ public void run() {
 			EntityType.FIREBALL,
 			// EntityType.FIREWORK,
 			// EntityType.FISHING_HOOK,
+			EntityType.FOX,
 			EntityType.GHAST, 
 			// EntityType.GIANT, 
 			EntityType.GUARDIAN, 
 			EntityType.HORSE, 
-			EntityType.HUSK,
```
Comment (C4K3) [inline src/PortalUtils.java:188]: There's a stray space at the end of this line.

File: src/PortalUtils.java [hunk 2]
```diff
@@ -205,12 +207,19 @@ public void run() {
 			EntityType.MUSHROOM_COW, 
 			EntityType.OCELOT, 
 			// EntityType.PAINTING,
+			EntityType.PANDA,
+			EntityType.PARROT,
+			EntityType.PHANTOM,
```
Comment (C4K3) [inline src/PortalUtils.java]: I don't think we want phantoms to be able to go to other worlds.

**Comment by C4K3** [comment]
Thanks for this!

There are a lot of lines with random spaces at the end, if you wouldn't mind fixing that :)

**Comment by zeroeightysix** [comment]
> Thanks for this!
> 
> There are a lot of lines with random spaces at the end, if you wouldn't mind fixing that :)

Whitespaces and phantoms removed.

**Pull request merged**